### ISSUE TITLE
Screen Reader is announcing incorrect info "list" is getting announced at Breadcrumb items.

### ISSUE THREAD

**Issue by huangzumeihan** [microsoft/fluentui#19214]
Actual:
Screen Reader is reading "list of 10 items" when the the focus lands at the breadcrumb.  screen reader announces incorrect information i.e "Out of list, main landmark, list with 10 itesm Link home", screen reader should not announce out of list. 

Expected:
The screen reader shouldn't announce "List" along with the deatils. 
Fabric's breadcrumb announces "List with 10 items"

![4](https://user-images.githubusercontent.com/31198399/127816589-a87fbacb-df19-4843-b0d3-addf6263602b.png)

**Comment by smhigley** [comment]
The Breadcrumb component is effectively a list of links (or buttons). It's announcing as a list by design, and list is a very common design choice for breadcrumbs (e.g. here: https://www.w3.org/TR/wai-aria-practices/examples/breadcrumb/index.html).

@huangzumeihan what's the rationale for saying this is incorrect?

**Comment by huangzumeihan** [comment]
This is the expected result of the tester
